driver-service/src/main/java/org/internship/task/driverservice/repositories/CarRepository.java

```diff
@@ -0,0 +1,12 @@
+package org.internship.task.driverservice.repositories;
+
+import org.internship.task.driverservice.entities.Car;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface CarRepository extends JpaRepository<Car,Long> {
+    Optional<Car> findByCarNumber(String email);
```

Comment:
по номеру или емейлу?
```suggestion
    Optional<Car> findByCarNumber(String number);
```